enzy_htp/analysis/rmsd.py

```diff
@@ -0,0 +1,120 @@
+#! python3
```

Comment:
In general, the line length in your code is longer than what we allow. See the google style guide for the reason

Reply:
Make it shorter than 100 characters (upper limit 140)